src/components/dashboard/Dashboard.js

```diff
@@ -12,8 +12,10 @@ class Dashboard extends Component {
         categories: [],
         chores: [],
         categoryName: '',
-        users: []
+        users: [],
+        editSaveBtn: true
```

Comment:
I think the variable name should be changed to make it easier to understand when you come back to the code later.  Maybe something like `editSaveBtnDisabled`